CHANGELOG.md

```diff
@@ -1,26 +1,75 @@
+See [documentation](https://docs.ferretdb.io/security/).
 
-There are no changes in the API, but since the data is stored in a different way, **this change is not backward compatible**.
-Please make a dump of your database before upgrading, delete the databases, upgrade FerretDB, and restore the dump afterwards.
+### New Features 🎉
+* Support $min field update operator by @chilagrow in https://github.com/FerretDB/FerretDB/pull/1652
+* Support `ordered` argument for `insert` command by @noisersup in https://github.com/FerretDB/FerretDB/pull/1673
+* Implement authentication for PostgreSQL by @AlekSi in https://github.com/FerretDB/FerretDB/pull/1725
+
+### Fixed Bugs 🐛
+* Fix unset document being updated by invalid value of $inc by @chilagrow in https://github.com/FerretDB/FerretDB/pull/1685
+
+### Enhancements 🛠
+* Update building documentation by @AlekSi in https://github.com/FerretDB/FerretDB/pull/1713
+
+### Documentation 📄
+* Add section for comparison and logical query operators by @Fashander in https://github.com/FerretDB/FerretDB/pull/1647
+* Add documentation for element query operators by @Fashander in https://github.com/FerretDB/FerretDB/pull/1675
+* Add documentation for array query operator by @Fashander in https://github.com/FerretDB/FerretDB/pull/1695
+* Enable blog post section by @Fashander in https://github.com/FerretDB/FerretDB/pull/1700
+
+### Other Changes 🤖
+* Simplify release procedure by @AlekSi in https://github.com/FerretDB/FerretDB/pull/1657
+* Modify `pjson` format by @rumyantseva in https://github.com/FerretDB/FerretDB/pull/1620
+* Bump deps by @AlekSi in https://github.com/FerretDB/FerretDB/pull/1664
+* Remove leading space from `SELECT` queries by @noisersup in https://github.com/FerretDB/FerretDB/pull/1665
+* Add `InTransactionRetry` helper by @rumyantseva in https://github.com/FerretDB/FerretDB/pull/1670
+* Add `mongo` test script example by @AlekSi in https://github.com/FerretDB/FerretDB/pull/1600
+* Use faster runner instances by @AlekSi in https://github.com/FerretDB/FerretDB/pull/1678
+* Update issue templates by @w84thesun in https://github.com/FerretDB/FerretDB/pull/1671
+* Bump Tigris version by @AlekSi in https://github.com/FerretDB/FerretDB/pull/1680
+* Move update tests to compat tests by @chilagrow in https://github.com/FerretDB/FerretDB/pull/1659
+* Add TODO comments by @AlekSi in https://github.com/FerretDB/FerretDB/pull/1687
+* Add `saslStart` stub by @w84thesun in https://github.com/FerretDB/FerretDB/pull/1649
+* Improve the way of storing data about collections by @rumyantseva in https://github.com/FerretDB/FerretDB/pull/1650
+* Implement `iterator.Interface` for `types.Document` and `types.Array` by @w84thesun in https://github.com/FerretDB/FerretDB/pull/1683
+* Improve issue template by @AlekSi in https://github.com/FerretDB/FerretDB/pull/1692
+* Remove $elemMatch and $slice projection operators by @chilagrow in https://github.com/FerretDB/FerretDB/pull/1698
```

Comment:
```suggestion
* Remove `$elemMatch` and `$slice` projection operators by @chilagrow in https://github.com/FerretDB/FerretDB/pull/1698
```

Reply:
"I ensured that the title is good enough for the changelog" is checked in that PR, and it is approved by you 😛

Reply:
PTAL

Reply:
> checked in that PR, and it is approved by you 😛

Yes, that's something to check more attentively in the future.